torch/quantization/fx/quantization_patterns.py

```diff
@@ -192,9 +192,23 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
                 else:
                     op = torch.ops.quantized.add
                 kwargs = {**self.binary_op_node.kwargs}
-                add_args = (*load_arg(quantized=True)(self.binary_op_node.args), scale_arg, zero_point_arg)
-                op = quantizer.quantized_graph.create_node(
-                    'call_function', self.quantized_binary_op, add_args, kwargs)
+                # not for land, just a hack to unblock for now
+                # TODO(before land): real fix
+                # add_args = (*load_arg(quantized=True)(self.binary_op_node.args), scale_arg, zero_point_arg)
+                # op = quantizer.quantized_graph.create_node(
+                    # 'call_function', self.quantized_binary_op, add_args, kwargs)
+                if True:
```

Comment:
We should figure out the best implementation here.  A challenge with the current approach is that there is a new type of arg, "potentially quantized".  So, we either need to increase the # of types of args from two to three, or think of a different approach to this PR.

Reply:
yeah this part looks hacky, is it possible to identify that a node is produced by ndim, size etc. and filter out these nodes during loading? something like:

```
int_node_idxs = [1]
all_arg_idxs = [1, 2]
quantized_idxs = [2]
... = load_arg(quantized=quantized_idxs)...
```